Kinvey/Kinvey/HttpRequest.swift

```diff
@@ -358,6 +358,16 @@ internal class HttpRequest: TaskProgressRequest, Request {
         if let clientAppVersion = client.clientAppVersion {
             request.setValue(clientAppVersion, forHTTPHeaderField: KinveyHeaderField.clientAppVersion)
         }
+        if let url = request.url,
+            let query = url.query,
+            query.contains("+"),
```

Comment:
@heyzooi Makes sense, but does the same thing have to happen for the other special URL characters:

`$-_.+!*'(),`